pudb/debugger.py

```diff
@@ -123,6 +123,7 @@
 
 Keys in stack list:
     enter - jump to frame
+    Ctrl-e - open file at line to edit with $EDITOR
```

Comment:
Oh I didn't notice that this is based on the stack list. I was expecting this to be implemented in the main code view.

Reply:
I understand why you think it would make more sense. I also thought about this and I implemented it here because of the resons:
- Maybe the developer would like to "look around" in the stack list for other files which are "effected" in the debugging. And it is easier with directly opening the files from stack list
- The `Ctrl-e` shortcut is useable/not allocated
- No (new) users can accidentaly open the file editor window. It takes 1 more shortcut to go to the stack list
- It is not confusing (even with the warning message) that the file is not changed which is in front of them after the edit.
- If we would open the current file which line would we jump to? The one under our cursor, the top/bottom/middle breakpoint? --> this can make some consusion, while on stack list the line number is displayed so no surprises will happen for the user

(maybe I am biased by my experience 😄 )

Reply:
FWIW, I also didn't catch that this was on the stack list. So there's at least two data points on where folks would expect this functionality to live. (I'm not saying remove it from the stack, but also add it to the main source window, too.) As for what line to show, I think the current highlight is the obvious choice.

Reply:
I fixed it, now both on Main view and Stack view the `Ctrl-e` edit functionality is working
@inducer Can you review it?